Cargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
 #    "./p1-staking-pool-dyn",
     "./p2-token-staking-fixed",
     "./p3-farm",
+    "./p3-farm-nft"
```

Comment:
this should be updated to match the directory name

Reply:
https://github.com/YellingOilbird/cheddar/blob/d2f1929e2f8ac966e7546e9732fa38aa17a49037/Cargo.toml#L9